src/styles/sidebar/components/TopBar.scss

```diff
@@ -25,38 +25,6 @@
   // the window is scrolled.
   transform: translate3d(0, 0, 0);
 
-  &__icon-button {
```

Comment:
This is all handled by `.CompactIconButton` styling now without the need for local override.